components/datetime/benches/fixtures/tests/patterns.json

```diff
@@ -0,0 +1,24 @@
+[
+    "dd/MM/y",
+    "dd/MM",
+    "d MMM",
+    "d MMM y",
+    "MMMM y",
+    "d MMMM",
+    "HH:mm:ss",
+    "HH:mm",
+    "y",
+    "mm:ss",
+    "h:mm:ss B",
+    "E, h:mm B",
+    "E, h:mm:ss B",
+    "E d",
+    "E h:mm a",
+    "y G",
+    "MMM y G",
+    "dd/MM",
+    "E, dd/MM",
+    "LLL",
+    "E, d MMM y",
+    "E, dd/MM/y"
```

Comment:
It would be nice if you could include some of the examples from zh, hi, etc., with multibyte characters.